### PR TITLE
dev-libs/libclc: Use correct LLVM_COMPAT version

### PR DESCRIPTION
**Pull request by FireBurn**
libclc 18 should be compiled against llvm 18

### CODE REVIEW

File: dev-libs/libclc/libclc-18.1.4.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-LLVM_COMPAT=( 17 )
+LLVM_COMPAT=( 18 )
```
Comment (mgorny) [inline dev-libs/libclc/libclc-18.1.4.ebuild:6]: Make it `17..18`. The only reasons older versions ain't here are limited keywords on spirv-llvm-translator.

**Review by mgorny — CHANGES_REQUESTED** [review]
Also set 19.0.0.9999 to the same.

**Comment by leio** [comment]
https://bugs.gentoo.org/931280

**Comment by mgorny** [comment]
I'm sorry, but this is taking too long. I'm going to fix it myself.

**Pull request closed without merging**